spec/amber/amber_spec.cr

```diff
@@ -55,8 +54,6 @@ describe Amber do
       end
 
       it "retains environment.yml settings that haven't been overwritten" do # NOTE: Any changes to settings here remain for all specs run afterwards.
-        # This is a problem.
```

Comment:
Until settings is change to an instances instead of singleton this is still an "a problem".